core/amber/src/main/scala/edu/uci/ics/texera/web/service/WorkflowExecutionService.scala

```diff
@@ -76,6 +71,13 @@ class WorkflowExecutionService(
   var executionConsoleService: ExecutionConsoleService = _
 
   def startWorkflow(): Unit = {
+    workflow = new WorkflowCompiler(workflowContext).compile(
+      request.logicalPlan,
+      resultService.opResultStorage,
+      lastCompletedLogicalPlan,
+      executionStateStore
+    )
+
```

Comment:
start workflow has a different semantic from `compile`. can you make them explicit?

Reply:
renamed to `executeWorkflow`